selfdrive/locationd/laikad.py

```diff
@@ -90,7 +90,9 @@ def load_cache(self):
   def cache_ephemeris(self, t: GPSTime):
     if self.save_ephemeris and (self.last_cached_t is None or t - self.last_cached_t > SECS_IN_MIN):
       put_nonblocking(EPHEMERIS_CACHE, json.dumps(
-        {'version': CACHE_VERSION, 'last_fetch_navs_t': self.last_fetch_navs_t, 'navs': self.astro_dog.navs},
+        {'version': CACHE_VERSION, 'last_fetch_navs_t': self.last_fetch_navs_t, 'navs': self.astro_dog.navs,
```

Comment:
do we need backwards compatibility? often, it's difficult to guarantee backwards compatibility, and it doesn't give you much anyway. usually it's easier to have robust detection for backwards incompatibility and bust the cache

Reply:
shane run into this, so it definitely might happen

i agree it doesnt give much,
what you mean with robust detection?
we have cache versioning but wasnt used back then, not sure how to handle differently